pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -107,7 +98,10 @@ export default function BlockComponentGallery() {
   }
 
   function handlePlayAgain() {
-    if (calculateWinner(gameState, showWinner) === true) {
+    calculateWin(gameState);
```

Comment:
You're not using the return value here?

Reply:
I deleted that line.  The original use for it is being taken care of by the code below.